### DISCUSSION TITLE
Allow Imlport live html and also Export pdf as Html and css

### DISCUSSION THREAD

**Discussion by davidbuckleyni** [QuestPDF/QuestPDF#854] [Ideas]
Other thing exist but none that intergrate well with quest pdf plz give some way of saving html version of the pdf.

Another part of this would be also allowing supply of html page and get a pdf produced by quest pdf.

**Comment by MarcinZiabek** [comment]
The closest thing I can provide is support for an additional SVG output format. Starting with the next major release, the library will produce not only raster images but also vector content, which (hopefully) will facilitate better integration with web pages. 

For injecting HTML content in QuestPDF documents, you can use the https://github.com/Relorer/HTMLToQPDF library. However, it does support only a subset of HTML features, making it excellent when integrating the output from WYSIWYG editors, but not entire modern websites.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
